src/NServiceBus.Metrics/CriticalTime/CriticalTimeMetricBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+using Metrics;
+using NServiceBus;
+using NServiceBus.Features;
+
+class CriticalTimeMetricBuilder : IMetricBuilder
+{
+    public void WireUp(FeatureConfigurationContext featureConfigurationContext, MetricsContext metricsContext, Unit messagesUnit)
+    {
+        var criticalTimeTimer = metricsContext.Timer("Critical Time", messagesUnit);
```

Comment:
@mikeminutillo - So in light of the above, I think you answered your own question:

> Are Timers the right way to go about exposing Critical Time?

By using the Metrics.Net timer and recording, we also get a min, max for the number of messages? Why would critical time be any different to the other metrics?